src/main/java/seedu/address/logic/Messages.java

```diff
@@ -64,13 +64,11 @@ public static String format(Person person) {
     public static String format(Job job) {
         final StringBuilder builder = new StringBuilder();
         builder.append(job.getName())
-                .append("; Company: ")
+                .append(" at ")
                 .append(job.getCompany())
-                .append("; Salary: ")
+                .append("\n")
                 .append(job.getSalary())
-                .append("; Requirements: ")
-                .append(job.getRequirements())
-                .append("; Description: ")
+                .append("\nDescription: ")
```

Comment:
While we are no longer using job requirement class, we use tag as its stead, is it something you would want to display here as well?

Reply:
I reverted this back with modification. I changed this originally to modify the Add Job Success message, but I realised it broke other commands that utilised this (i.e. match command) so I simply shifted the changed message to addJobCommand.

Reply:
Sure, so we are still printing our the tags (requirement) in formatting?